### PR TITLE
NO-JIRA: bringing back the server-side apply test

### PR DESCRIPTION
**Pull request by deads2k**
/assign @bertinatto 

@stbenjam which specific payload test would you like?

### CODE REVIEW

File: pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go [hunk 1]
```diff
@@ -55,42 +55,70 @@ func (*auditLogAnalyzer) ConstructComputedIntervals(ctx context.Context, startin
...
+
+		failures := []string{}
+		flakes := []string{}
+		for username, numberOfApplies := range usersToApplies {
+			if numberOfApplies > 200 {
+				switch username {
+				case "system:serviceaccount:openshift-infra:serviceaccount-pull-secrets-controller",
+					"system:serviceaccount:openshift-network-operator:cluster-network-operator",
+					"system:serviceaccount:openshift-infra:podsecurity-admission-label-syncer-controller",
+					"system:serviceaccount:openshift-monitoring:prometheus-operator":
```
Comment (bertinatto) [inline pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go:75]: xref: https://prow.ci.openshift.org/view/gs/test-platform-results/logs/openshift-kubernetes-2086-nightly-4.18-e2e-aws-ovn-single-node-techpreview/1836393451122331648

`cluster-olm-operator` issued 1424 patch requests.

**Comment by deads2k** [comment]
green payloads: https://pr-payload-tests.ci.openshift.org/runs/ci/938f8ef0-75c9-11ef-94ff-0544af29b405-0

**Pull request merged**